packages/sdk-components-animation/vitest.config.ts

```diff
@@ -1,6 +1,30 @@
 import { defineConfig } from "vitest/config";
+import { existsSync, readdirSync } from "node:fs";
+import path from "node:path";
+import fg from "fast-glob";
+
+const rootDir = ["..", "../..", "../../.."]
```

Comment:
no other way?

Reply:
Shortest way to find folder with `.git` folder inside.
There are packages exists to find root folder, but like 1mb in node_modules